src/main/java/seedu/address/model/util/ModuleBuilder.java

```diff
@@ -0,0 +1,101 @@
+package seedu.address.model.util;
+
+import seedu.address.model.module.Code;
+import seedu.address.model.module.Credit;
+import seedu.address.model.module.Grade;
+import seedu.address.model.module.Module;
+import seedu.address.model.module.Semester;
+import seedu.address.model.module.Year;
+
+/**
+ * A utility class to help with building Module objects.
+ */
+public class ModuleBuilder {
```

Comment:
Is there a reason why you moved this to the main code? I believe I've already written this same piece of code under test because its only used in test.

Reply:
Ah, I wanted to use it in the main app, to help produce the initial data (similar to how AB4 already has some data when you run, otherwise that part will be a bit tedious. Because it's in test folder I didn't know how to import it (also it seems to me that we should keep `src` files independent of `test` files), so I just duplicated it. Perhaps it makes more sense to just have it in main app, and our tests import it from there?

Reply:
If that's the case then the one in test should be deleted and test that relies on this class should reference this instead.

Reply:
Got it. Will do tmr.